sources/packages/web/src/views/aest/student/StudentRestrictions.vue

```diff
@@ -6,14 +6,18 @@
       :canResolveRestriction="true"
     ></student-restrictions
   ></tab-container>
+  <tab-container>
```

Comment:
Do you think it will be a good idea to create a new component(something like Restrictions and Scholastic Standing) which combines restriction and scholastic standing history so that the same one component can be shared between ministry and student?

Reply:
I would say that would be nice to have but since we have components for both containers I would recommend not investing time on it right now.